components/gc-features/gc-features-doc-en.html

```diff
@@ -0,0 +1,637 @@
+---
+title: "Context-specific features - Canada.ca design pattern"
+description: "Documentation for the Context-specific features."
+language: "en"
+altLangPage: "gc-features-doc-fr.html"
+secondlevel: false
+dateModified: "2022-10-21"
+share: "true"
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+
+	<dt>Last review</dt>
+	<dd>2021-07-21</dd>
+
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/common-design-patterns/feature-tiles.html">Context-specific features - Canada.ca design pattern</a></dd>
+
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>gc-features</code></li>
+		</ul>
+	</dd>
+
+	<dt>Component version</dt>
+	<dd>Version 5.0</dd>
+
+	<dt>Supported working versions</dt>
+	<dd>
+		<ul>
+			<li>Version 5</li>
+			<li>Version 4</li>
```

Comment:
In this page, we are missing the reference that say our version 4 of the gc-features was released with GCWeb version 9.5.0